### PR TITLE
AMD Pervasive AI Contest blog

### PR DESCRIPTION
**Pull request by mfuntowicz**

### CODE REVIEW

**Review by fxmarty — APPROVED** [review]
LGTM

File: amd_pervasive_developer_ai_contest.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+authors:
+- user: guruprasadmp
+---
+
+# AMD Pervasive AI Developer Contest
+
+AMD and Hugging Face are actively engaged in helping developers seamlessly deploy cutting-edge AI models on AMD hardware. 
+This year, AMD takes their commitment one step further by providing developers free, hands-on access to state-of-the-art AMD hardware through their recently announced [Pervasive AI Developer Contest](https://www.hackster.io/contests/amd2023#challengeNav). 
+This global competition is an incubator of AI innovation, beckoning developers worldwide to create unique AI applications. 
+Developers can choose from three exciting categories: Generative AI, Robotics AI, and PC AI and compete for cash prizes up to $10,000 USD.
```
Comment (guruprasad-mp) [inline amd_pervasive_developer_ai_contest.md:13]: Can we say, $10,000 per category and a total of $160,000 prizes being given away.

File: _blog.yml [hunk 2]
```diff
@@ -3448,4 +3448,13 @@
     - guide
     - llm
     - nlp
-    - tgi
\ No newline at end of file
+    - tgi
+
+- local: amd_pervasive_developer_ai_contest
```
Comment (julien-c) [inline _blog.yml:3453]: we use dashes not underscores usually

File: _blog.yml [hunk 3]
```diff
@@ -3448,4 +3448,13 @@
...
+  title: "AMD Pervasive AI Developer Contest!"
+  author: guruprasadmp
+  thumbnail: /blog/assets/amd_pervasive_developer_ai_contest/amd_developer_general_abstract.jpg
```
Comment (julien-c) [inline _blog.yml:3456]: no big deal but note that thumbnail is not in correct aspect ratio

**Pull request merged**